Sources/streams/DuplexStream.swift

```diff
@@ -243,20 +243,21 @@ private class _DuplexReadStream<TI, TO> : ReadableStream<TI> {
   }
   
   override func _primaryRead(count howMuchToRead: Int) {
-    parent._primaryRead(count: howMuchToRead)
+    parent!._primaryRead(count: howMuchToRead)
```

Comment:
If you really want that behaviour, do a `private var parent : DuplexStream<TI, TO>!`. Otherwise do a proper guard with a descriptive fatalError (`guard let p = parent else { fatalError("parent of duplex stream is unexpectedly nil?!") }`)